src/sc/src/controllers/partitions/controller.rs

```diff
@@ -40,7 +40,6 @@ impl PartitionController {
         spawn(controller.dispatch_loop());
     }
 
-    #[instrument(skip(self), name = "PartitionController")]
```

Comment:
Shouldn't keep this? that way we stat with same root

Reply:
The problem is that the dispatch_loop is a function that "never" ends, since it loops. This would mean that we have one huge root span for each controller that never ends, which would actually make it very difficult to see what is happening. I think the proper "root" span here is the top-level function that gets called inside this loop, i.e. the handler.

Reply:
make sense